nav.php

```diff
@@ -15,25 +15,25 @@
 	</a>
 	<div class="navbar-links">
 		<ul>
-			<li class="hover-underline-animation"><a href="index.php">Home</a></li>
-			<li class="hover-underline-animation"><a href>Playlists</a>
+			<li><a href="index.php">Home</a></li>
+			<li><a href>Playlists</a>
 				<ul>
-					<li class="hover-underline-animation"><a href="query1.php">Query 1</a></li>
-					<li class="hover-underline-animation"><a href="query2.php">Query 2</a></li>
+					<li><a href="query1.php">Query 1</a></li>
+					<li><a href="query2.php">Query 2</a></li>
 				</ul>
 				<?php
 					//if the admin logs in (Graeme) then it the nav will show an additional link to the admin settings pages
                     if ($_SESSION["login_user"] == "Graeme"){
-                    	echo"<li class='hover-underline-animation'><a href>Admin Settings</a>
```

Comment:
#removed underline animation as I added a text shadow feature for all links instead